pages/open_cas_linux/guide_product_specs_requirements.md

```diff
@@ -0,0 +1,95 @@
+| Storage          | Primary storage devices: Any device that appears as a local block device. For example, local disk, RAID, SAN iSCSI (via Fibre Channel, Infiniband, Ethernet), etc. Core device (HDD) logical block size must be 512 bytes or larger. Operating system must be on separate partition or drive than the core storage device to be cached. |
+| File Systems     | The following file systems are supported for primary storage or core storage:                                                                                                                                                                                                                                                           |
+| Software         | The following prerequisite software must be installed prior to Open CAS Linux installation:                                                                                                                                                                                                                                                  |
+| Power Management | Power Management Suspend (S3) and Hibernate (S4) states must be disabled.                                                                                                                                                                                                                                                               |
+
+-   Intel® SSD DC S3700 Series
+
+-   Intel® SSD DC P3700 Series
+
+Cache device logical block size must be smaller than or equal to the logical
+block size of the core storage device. 60GB or larger capacity is recommended.
+
+**NOTE:** Older firmware on some SSDs lose their format after reboot and cause
+data loss. We highly recommend that you upgrade to the latest recommended
+firmware and boot loader versions for your devices. For example, as of the
+writing of this document, the firmware versions (8DV101H0 for firmware &
+8B1B0133 for bootloader) should be used for Intel SSD DC P3700 Series drives.
+
+-   ext3 (limited to 16TiB volume size). This is an ext3 file system limitation.
+
+-   ext4
+
+-   xfs
+
+**NOTE:** For best performance, core device file system block size should match
+the core device partition logical block size.
+
+**NOTE:** For best performance, IO request sizes in multiples 4KiB are
+recommended.
+
+-   bzip2
+
+-   tar
+
+-   sed
+
+-   make
+
+-   gcc
+
+-   kernel-devel
+
+-   kernel-headers
+
+-   python2
+
+-   lsblk
+
+-   lsof
+
+-   argparse (python module)
+
+The following software is optional. If desired, it must be installed prior to
+Open CAS Linux installation:
+
+-   dkms (<http://linux.dell.com/dkms/>)
```

Comment:
DKMS support is not available in OpenCAS. Please remove.

Reply:
I will fix this